lib/rack/response.rb

```diff
@@ -36,7 +36,7 @@ def self.[](status, headers, body)
     # Initialize the response object with the specified body, status
     # and headers.
     #
-    # @param body [nil, #each, #to_str] the response body.
+    # @param body [nil, #each, #call, #to_s] the response body.
```

Comment:
This should probably be `to_str` and no be changed to `to_s`, since that's the method the code checks for.

Reply:
Ohhh my bad, thanks for catching that.

Reply:
```suggestion
    # @param body [nil, #each, #call, #to_str] the response body.
```